scripts/zombienet/native/local-testnet.toml

```diff
@@ -2,31 +2,41 @@
 timeout = 1000
 provider = "native"
 
+# Using Rococo 1.7.0 as relay.
 [relaychain]
-default_command = "polkadot"
+default_command = "../polkadot-sdk/target/release/polkadot"
```

Comment:
Could we add a bin folder that is ignored by git where we can personally add these binaries?

Reply:
Sure!

Reply:
Addressed and tested in https://github.com/Polimec/polimec-node/pull/217/commits/82f9d3f56dc25fd0654950924e97c6563cd4ea98. 

Should also the polimec-node point to the bin folder or should I leave it to` ./target/release`?